hashes/src/lib.rs

```diff
@@ -257,7 +259,7 @@ pub trait Hash:
     fn from_byte_array(bytes: Self::Bytes) -> Self;
 
     /// Copies a byte slice into a hash object.
-    #[deprecated(since = "0.15.0", note = "use `from_byte_array` instead")]
```

Comment:
Why was this removed? It needs to stay.

Reply:
Ooph, I probably flicked to the wrong buffer. I was adding and removing attributes running the linter. I'll take another look. Thanks. FTR no deprecation attribute on trait method impls.